examples/multitenancy/appday2ops/web-app/docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: "3"
```

Comment:
Since we have a docker-compose file (probably coming from the link that you have in notes.txt), it will be good to have the docker-compose command in steps.txt if someone wants to try the example locally without minikube.